docs/concepts/services/input-pane.md

```diff
@@ -0,0 +1,74 @@
+---
+id: input-pane
+title: Input Pane
+---
+
+# Input Pane <MinVersion version="11.1" />
+
+The `InputPane` allows developers to listen for the platform's input pane (e.g., software keyboard or on-screen keyboard) current state and boundaries.
+
+The `InputPane` can be access through an instance of `TopLevel` or `Window`, for more details on accessing `TopLevel` please visit [TopLevel](../toplevel) page:
+```cs
+var inputPane = TopLevel.GetTopLevel(control).InputPane;
+```
+
+:::note
+Currently, Avalonia does not* automatically adjust root view and scrolling position depending on state of the input pane. Instead, it is recommended for developers to use IInputPane API and adjust their apps accordingly.
```

Comment:
Was "not*" intentional or a typo / missing markup italic?
input pane -> `InputPane`, if we're using proper nouns. A few other occurences in the doc.
Need backticks around IInputPane, too.

Reply:
It was, since I added details below:
```markdown
\* Automatic adjustment is planned for future 11.* releases.
```
Will remove that anyway, will keep as two paragraphs instead.

Reply:
> input pane -> InputPane

Almost everywhere "input pane" is used, I copied from the Avalonia xml docs. I would keep it consistent in case if we auto-generate these one day.